crypto/crypto/Cargo.toml

```diff
@@ -36,10 +36,12 @@ tiny-keccak = { version = "2.0.2", features = ["sha3"] }
 vanilla-x25519-dalek = { version = "1.1.0", package = 'x25519-dalek', default-features = false, features = ["std"], optional = true }
 x25519-dalek = { git = "https://github.com/novifinancial/x25519-dalek.git", branch = "fiat5", version = "1.1.0", default-features = false, features = ["std", "fiat_u64_backend"], optional = true}
 aes-gcm = "0.8.0"
-diem-crypto-derive = { path = "../crypto-derive", version = "0.1.0" }
+
 bcs = "0.1.2"
+diem-crypto-derive = { path = "../crypto-derive", version = "0.1.0" }
 diem-nibble = { path = "../../common/nibble", version = "0.1.0" }
 diem-workspace-hack = { path = "../../common/workspace-hack", version = "0.1.0" }
+fallible = { path = "../../common/fallible", version = "0.1.0" }
```

Comment:
So one thing i'm working on is being able to publish the crypto crate and I would really like if we could avoid adding *another* dep to this list that needs to be considered published to crates.io.

Reply:
ah interesting, is fallible the only intruder here? Also what's the reason for publishing the crypto crate : o ? It currently mostly contains wrappers

Reply:
It's not only a wrapper? But it's true that many of our updates re key security have been accepted from Dalek authors to upstream, a few of them re malleability not yet though.

Reply:
good point! I'm wondering if these will eventually make it to our fork of dalek :P